eapp/public/js/acquisitionform.js

```diff
@@ -220,15 +223,47 @@ function add_term_to_form(termform){
         }
       }
     }else{
-
-      $("#ndar-fields").append('<div class="form-group">\
+      $("#ndar-fields").append('<div class="form-group" id="ndar-'+i+'x">\
       <label for="ndar-'+i+'" data-toggle="tooltip" title="'+selectedFields[i].name+'">'+selectedFields[i].description+'</label>\
         <div>\
           <input class="form-control" type="text" placeholder="'+selectedFields[i].valueRange+'" id="ndar-'+i+'">\
         </div>\
       </div>')
     }
 
+    if(document.getElementById('ndar-'+i+'') != null){
+      document.getElementById('ndar-'+i+'').onblur = function(){fieldValidation()}
+    }
+
+    /*
+    Validates input type is correct
+    So far only for numbers, strings, and dates
+    */
+    function fieldValidation() {
+      let integer = selectedFields[i].type == "Integer"
+      let string = selectedFields[i].type == "String"
+      let date = selectedFields[i].type == "Date"
+      let errorMessage = '<div class="alert alert-danger alert-dismissible" role="alert">\
+          <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>\
+          <strong>Error!</strong> Input is incorrect data type.\
```

Comment:
Provide hint of the data type (or range of values) required. For example, if a user enters integer number to a string field, provide hint as string is required. If user enters an integer out of range, e.g., age has to be in the range (0-1260), appropriate hint should be provided.